docs/download.md

```diff
@@ -65,14 +68,13 @@ for your hardware's minimum driver version.
 ### RAPIDS Accelerator's Support Policy for Apache Spark
 The RAPIDS Accelerator maintains support for Apache Spark versions available for download from [Apache Spark](https://spark.apache.org/downloads.html)
 
-### Download RAPIDS Accelerator for Apache Spark v23.12.2
-- **Scala 2.12:**
-  - [RAPIDS Accelerator for Apache Spark 23.12.2 - Scala 2.12 jar](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.12/23.12.2/rapids-4-spark_2.12-23.12.2.jar)
-  - [RAPIDS Accelerator for Apache Spark 23.12.2 - Scala 2.12 jar.asc](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.12/23.12.2/rapids-4-spark_2.12-23.12.2.jar.asc)
-
-- **Scala 2.13:**
-  - [RAPIDS Accelerator for Apache Spark 23.12.2 - Scala 2.13 jar](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.13/23.12.2/rapids-4-spark_2.13-23.12.2.jar)
-  - [RAPIDS Accelerator for Apache Spark 23.12.2 - Scala 2.13 jar.asc](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.13/23.12.2/rapids-4-spark_2.13-23.12.2.jar.asc)
+### Download RAPIDS Accelerator for Apache Spark v24.02.0
+| Processor | Scala Version | Download Jar | Download Signature |
+|-----------|---------------|--------------|--------------------|
+| x86_64    | Scala 2.12    | [RAPIDS Accelerator 2.12 v24.02.0](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.12/24.02.0/rapids-4-spark_2.12-24.02.0.jar) | [Signature](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.12/24.02.0/rapids-4-spark_2.12-24.02.0.jar.asc) |
+| x86_64    | Scala 2.13    | [RAPIDS Accelerator 2.13 v24.02.0](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.13/24.02.0/rapids-4-spark_2.13-24.02.0.jar) | [Signature](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.13/24.02.0/rapids-4-spark_2.13-24.02.0.jar.asc) |
+| arm64     | Scala 2.12    | [RAPIDS Accelerator 2.12 v24.02.0](#) | [Signature](#) |
+| arm64     | Scala 2.13    | [RAPIDS Accelerator 2.13 v24.02.0](#) | [Signature](#) |
```

Comment:
These links need to be fixed

Reply:
Yes, I'll keep that in mind. I don't know what these links are going to be as it's the first release. I'll update them in a different PR when we update gh-pages.

Reply:
Check with @NvTimLiu, these should be flled out here, not hacked into gh-pages afterwards unless we're for some reason not releasing ARM binaries simultaneously?

Reply:
Sure!

Reply:
I think the arm jars will be include the cuda classifier, and will be named 

rapids-4-spark_2.12-24.04.0-cuda11-arm64.jar
rapids-4-spark_2.12-24.04.0-cuda12-arm64.jar

rapids-4-spark_2.13-24.04.0-cuda11-arm64.jar
rapids-4-spark_2.13-24.04.0-cuda12-arm64.jar

Related issue: https://github.com/NVIDIA/spark-rapids/issues/6881

Reply:
Drafted [10414](https://github.com/NVIDIA/spark-rapids/pull/10414) to fix this, thanks!